src/go/k8s/tests/e2e/update-image-tls-client-auth-external-ca/03-assert.yaml

```diff
@@ -0,0 +1,158 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ca-cert
+
+---
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: client-cert-key
+
+---
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: update-img-external-client-ca-schema-registry-trusted-client-ca
+
+---
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: update-img-external-client-ca-proxy-api-trusted-client-ca
```

Comment:
I'm not sure what we are asserting here. That there is secret with specific name. Shouldn't we validate if those certificates are used by Redpanda using openssl cli?

Reply:
Removed ca-cert and client-cert-key since they are done with TestStep. The last two update-img-external-client-ca-schema-registry-trusted-client-ca and update-img-external-client-ca-proxy-api-trusted-client-ca are the secret created by the operator to store the bundled CAs, so they assert that the operator has created those.

See my [response](https://github.com/redpanda-data/redpanda/pull/9863#discussion_r1169195668) regarding that it is not viable to verify with openssl. The tests ultimately relies on curl calls to verify whether the CA configuration has been consumed by RP.